upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -98,6 +100,9 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap) {
 	dest["EncodeGCELabel"] = gce.EncodeGCELabel
 
 	dest["DnsControllerImage"] = tf.DnsControllerImage
+
+	// TODO: Only for AWS?
+	dest["AWSRegion"] = tf.AWSRegion
```

Comment:
The built in template function 'Region' already sets the region properly based on which cloud provider to use. I'm going to remove this change and use that one instead.